<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1200</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>700</height>
    </size>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_m">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="new_a"/>
    <addaction name="open_a"/>
    <addaction name="save_a"/>
    <addaction name="saveas_a"/>
    <addaction name="cloud_a"/>
   </widget>
   <widget class="QMenu" name="setting_m">
    <property name="title">
     <string>设置(&amp;P)</string>
    </property>
    <addaction name="write_a"/>
   </widget>
   <addaction name="file_m"/>
   <addaction name="setting_m"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="new_a">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="open_a">
   <property name="text">
    <string>打开项目(&amp;O)</string>
   </property>
  </action>
  <action name="save_a">
   <property name="text">
    <string>保存项目(&amp;S)</string>
   </property>
  </action>
  <action name="saveas_a">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="write_a">
   <property name="text">
    <string>编辑(&amp;V)</string>
   </property>
  </action>
  <action name="cloud_a">
   <property name="text">
    <string>上传到云端(&amp;C)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
